File: js/number3.js
const readlineSync = require('readline-sync');

function cntNonDecSeries() {
    let len_series = 0;
    let cnt = 0;

    const len = parseInt(readlineSync.question(''), 10) || 0;

    if (len > 0) {
        let num1 = parseInt(readlineSync.question(''), 10);
        let num2 = 0;

        for (let i = 1; i < len; i++) {
            num2 = parseInt(readlineSync.question(''), 10);

            if (num2 >= num1) {
                len_series++;
            } else if (len_series !== 0) {
                cnt++;
                len_series = 0;
            }
            num1 = num2;
        }

        if (len_series !== 0) {
            cnt++;
        }
    }
    return cnt;
}

const result = cntNonDecSeries();
console.log(`Количество неубывающих серий: ${result}`);

File: swift/number3.swift
import Foundation

func cntNonDecSeries() -> Int {
    var lenSeries = 0
    var cnt = 0

    let len = Int(readLine()!)!

    if len > 0 {
        var num1 = Int(readLine()!)!
        var num2 = 0

        for _ in 1..<len {
            num2 = Int(readLine()!)!

            if num2 >= num1 {
                lenSeries += 1
            } else if lenSeries != 0 {
                cnt += 1
                lenSeries = 0
            }
            num1 = num2
        }

        if lenSeries != 0 {
            cnt += 1
        }
    }
    return cnt
}

let result = cntNonDecSeries()
print("Количество неубывающих серий: \(result)")